server/game/cards/19-JS/Harlaw.js

```diff
@@ -0,0 +1,26 @@
+const DrawCard = require('../../drawcard.js');
+
+class Harlaw extends DrawCard {
+    setupCardAbilities(ability) {
+        this.persistentEffect({
+            match: (card) => card === card.controller.activePlot,
+            targetController: 'opponent',
+            effect: ability.effects.modifyReserve(-1)
```

Comment:
With all of the potential reserve reducers now, I had to double check that we don't allow it to go below 0. 😅